src/domain/individualCustomer/IndividualCustomerPageContainer.tsx

```diff
@@ -13,10 +13,10 @@ import CardBody from '../../common/cardBody/CardBody';
 import BillsCard from './billsCard/BillsCard';
 import CustomerInfoCard from '../cards/customerInfoCard/CustomerInfoCard';
 import LoadingSpinner from '../../common/spinner/LoadingSpinner';
-import ApplicationCard from './applicationCard/ApplicationCard';
+import ApplicationCard from './applicationsCard/ApplicationsCard';
```

Comment:
Here you might consider changing the name to `ApplicationsCard`.

Reply:
Thanks! I made a commit fixing this